src/oop/static_modifier/hard/q5/QuestionText.md

```diff
@@ -0,0 +1,34 @@
+# oop-static_modifier-hard-q5
+
+- Question Type: Java question
+
```

Comment:
Carpet squares are easy to order but the colour of different batches can be slightly different and so cannot be used for the same carpet. Write a Java class called `Carpet` to keep track of the carpet squares of the most recent order using the requirements below. Then write a class called `TestCoverage` to test `Carpet`.